x/incentives/keeper/distribute.go

```diff
@@ -20,22 +21,28 @@ func (k Keeper) getDistributedCoinsFromGauges(gauges []types.Gauge) sdk.Coins {
 	return coins
 }
 
+// getToDistributeCoinsFromGauges returns coins that have not been distributed yet from the provided gauges
 func (k Keeper) getToDistributeCoinsFromGauges(gauges []types.Gauge) sdk.Coins {
 	// TODO: Consider optimizing this in the future to only require one iteration over all gauges.
+	// TODO: Lets make issue for this, requires changing getGaugesFromIterator logic in gauge.go
 	coins := k.getCoinsFromGauges(gauges)
 	distributed := k.getDistributedCoinsFromGauges(gauges)
 	return coins.Sub(distributed)
 }
 
+// getToDistributeCoinsFromIterator utilizes db.iterator to return a list of gauges.
```

Comment:
nit: just a thought that it might be better to refer to identifier names (`iterator`) instead of type names (`db.Iterator`) in specs. This way, we do not depend on the external package - if the type name changes, the comment still make sense
```suggestion
// getToDistributeCoinsFromIterator utilizes iterator to return a list of gauges.
```